apps/tlon-mobile/src/hooks/useNotificationListener.ts

```diff
@@ -56,13 +53,17 @@ export default function useNotificationListener({
             },
           },
         } = response;
+        const postId = getPostIdFromWer(data.wer);
```

Comment:
Good call. I moved the helpers to the api layer. 

I have low confidence this isn't changing soon, the backend plan is to have `%notify` handle this information rather than `%hark`. It's also a bit of a grab bag in terms of what the Swift helper makes semantic vs leaves urbitty rn. Between those two factors, I'd say lets hold off on specifying more rigorous parsing/interfaces for the moment.